<launch>

	<!-- Load the parameters file -->
	<rosparam command="load" file="$(find robonuc)/params/default.yaml"/>

	<node name="hokuyo0" pkg="hokuyo_node" type="hokuyo_node">
		<param name="port" type="string" value="/dev/ttyACM0" />
		<param name="frame_id" value="/laser0"/>
		<param name="min_ang" type="double" value="-1.9" />
		<param name="max_ang" type="double" value="1.9" />
		<param name="skip" type="int" value="0" />
		<param name="intensity" value="false" />
		<remap from="scan" to="scan0" />
	</node>
	<node name="hokuyo1" pkg="hokuyo_node" type="hokuyo_node">
		<param name="port" type="string" value="/dev/ttyACM1" />
		<param name="frame_id" value="/laser1"/>
        	<param name="min_ang" type="double" value="-2.3" />
        	<param name="max_ang" type="double" value="2.3" />
		<param name="skip" type="int" value="0" />
		<param name="intensity" value="false" />
		<remap from="scan" to="scan1" />
	</node>
	<node pkg="robonuc" type="vs_laser_receptor" name="vs_laser_receptor" output="screen"/>
	<node pkg="robonuc" type="vs_platform_sim" name="vs_platform_sim" output="screen"/>
	<node pkg="robonuc" type="vs_frame" name="vs_frame" output="screen"/>

</launch>
